package.json

```diff
@@ -12,7 +12,7 @@
     "build": "run-s bootstrap build:**",
     "build:js": "webpack --config webpack.config.js --display-error-details",
     "build:app": "webpack --config webpack.example.app.js --display-error-details",
-    "watch:js": "chokidar 'src/**/*.jsx' 'example/*.js' 'webpack.config.js' -c 'npm run build:**'",
+    "watch:js": "chokidar 'src/**/*.jsx' 'example/*.js' 'webpack.config.js' -c 'run-s build:**'",
```

Comment:
actually that should just be `npm run build`, since build runs `bootstrap` (which is pretty necessary), and then falls through to build:**